notebooks/Airline Delays - Final Writeup

```diff
@@ -234,11 +288,66 @@ ORDER BY origin_aa1_rain_depth"""))
 
 # COMMAND ----------
 
+# MAGIC %md
+# MAGIC ####Airline Flight Feature Engineering
+
+# COMMAND ----------
+
+
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC ####Weather Feature Engineering
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC To identify weather patterns the could cause delays, we filtered our training data to look at flights that had a weather delay and did not have a weather delay to understand what types of weather thresholds could be used to created indicator variables.  After reviewing the summary statistics for the weather variables, we decided to create four weather indicator variables to increase the signal we might see from the weather features in our models.  The first feature we created was focused on identifying flights that had very low ceiling height below 2,000.  Per the summary statisitic tables below, the average origin ceiling height for weather delays is 4,000 below the ceiling height for non weather delay flights. Moreover, we also noticed the origin wind speed rate is higher for weather delayed flights so we created an indicator that highlights if wind is above 50.  The last two features we created focused on winter and summer temperature trends.  Per the origin air temperature histograms below, the weather delayed flights had a higher percentage of summer flights with below air temperature values below 0, so we created a indicator variable to identify these types of weather patterns during the summer. 
```

Comment:
"weather thresholds could be used to created indicator variables" - typo "created" -> "create"